bop_toolkit_lib/tests/eval_bop24_pose_test.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+import subprocess
+import os
+import re
+import time
+import numpy as np
+from tqdm import tqdm
+import argparse
+from bop_toolkit_lib import inout
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--num_false_positives", default=0, type=int)
+args = parser.parse_args()
+
+# Define the input directory
+INPUT_DIR = "./bop_toolkit_lib/tests/data/"
+
+# Define the output directory
+OUTPUT_DIR = "./bop_toolkit_lib/tests/logs"
+os.makedirs(OUTPUT_DIR, exist_ok=True)
+
+# Define the dataset dictionary
+FILE_DICTIONARY = {
+    # "icbin": "cnos-fastsammegapose_icbin-test_7c9f443f-b900-41bb-af01-09b8eddfc2c4.csv",
+    # "tudl": "cnos-fastsammegapose_tudl-test_1328490c-bf88-46ce-a12c-a5e5a7712220.csv",
+    "lmo": "cnos-fastsammegapose_lmo-test_16ab01bd-f020-4194-9750-d42fc7f875d2.csv",
+    # "ycbv": "cnos-fastsammegapose_ycbv-test_8fe0af14-16e3-431a-83e7-df00e93828a6.csv",
+    # "tless": "cnos-fastsammegapose_tless-test_94e046a0-42af-495f-8a35-11ce8ee6f217.csv",
+    # Add more entries as needed
+}
```

Comment:
Could we maybe run a last sanity test and check whether we get 0mAP with empty predictions and 100mAP with ground truth predictions?

Reply:
Yes, I agree this is important. I added in my latest commit the sanity test with ground-truth predictions. 

For empty predictions, maybe you mean “totally wrong” predictions instead of “empty” since the current code will already raise some errors when the predictions are empty anyway.